_pytest/compat.py

```diff
@@ -253,6 +253,29 @@ def safe_str(v):
             return v.encode('ascii', errors)
 
 
+COLLECT_FAKEMODULE_ATTRIBUTES = (
+    'Collector',
+    'Module',
+    'Generator',
+    'Function',
+    'Instance',
+    'Session',
+    'Item',
+    'Class',
+    'File',
+    '_fillfuncargs',
+)
+
+
+def _setup_collect_fakemodule():
```

Comment:
Just to be clear, just creating `_pytest/collect.py` which imports those simbols, and add a line `from _pytest import collect` in `pytest.py` breaks Python2.6? Just trying to clarify in case I didn't make my point across.